recipes/arrow/all/patches/11.0.0-0001-fix-cmake.patch

```diff
@@ -0,0 +1,42 @@
+diff --git a/cpp/cmake_modules/FindThriftAlt.cmake b/cpp/cmake_modules/FindThriftAlt.cmake
+index f3e49021d..95177c2a6 100644
+--- a/cpp/cmake_modules/FindThriftAlt.cmake
++++ b/cpp/cmake_modules/FindThriftAlt.cmake
+@@ -45,22 +45,21 @@ endif()
+ #   * https://github.com/apache/thrift/pull/2725
```

Comment:
A good practice is to put this links in the `patch_source`, see https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/conandata_yml_format.md#patch_source

Reply:
None of these links are patch sources.
These are just some of the issues mentioning problems related to finding Thrift that I stumbled upon while looking for existing solutions to the build failure I was debugging.